careers/jobs/senior-fullstack-node-engineer.md

```diff
@@ -0,0 +1,65 @@
+
+At DVELP we believe in transparency, so all our business processes, information security, and coding principles are publicly accessible - to learn more about our approach check out [our cookbook](../..//Readme.md).
+
+## Key Responsibilities of the Role
+
+- Lead and advise on entire projects
+- Be hands-on in new and existing code bases
+- Working with the existing team to set and maintain standards of code quality and best practice across the organisation
+- Work closely with the product team to understand, scope and advise on the best approach to achieve a business objective
+- Work with client’s engineering teams; leading, advising, and contributing to their solutions
+
+## Day to Day Tasks
+
+- Discuss and design new systems from the ground-up with strong opinions about software and design principles
+- Develop and maintain new and existing systems
+- Write documentation to record and explain decisions around architecture and design
+- Write unit integration tests as a first-class citizen
+- Perform code reviews and participate actively in discussions, even around technologies you may not be an expert in
+
+## About You
+
+- Experienced and comfortable through the entire lifecycle of a feature, from ideation to delivery
+- Can understand business drivers and make solid proposals for building new features or refining existing ones
+- Are a subject matter expert in at least one programming environment
+- Have typically 5-8 years experience in a specific programming domain or language
+- Excellent communications skills (native or strong English speaker)
+- Experienced working directly with clients or key stakeholders
+- Are ambitious and excited to grow with DVELP
+- Must be willing to travel to client locations when required
+
+### Technical Skills
+
+While we understand that good programmers can learn new skills, this role is ideally for someone who has good working knowledge and experience in the following:
+
+- React
+- Redux
+- Typescript
+- Node (Express)
+- GraphQL
+
+More generally, desirable but not required are:
+
+- Experience with Twilio APIs
+- Experience with Google AI APIs
+
+We know there's a lot more to programming than what's listed above - but we'd rather learn through talking where your strengths are, than prescribe a pre-set list of hard requirements.
+
+## What Next?
+
+We are looking for someone to start as soon as possible, so applications will be considered as and when we receive them.
+
+DVELP is a remote-first company, so working from home or a co-working space is your day-to-day. DVELP (and its parent group Sabio) do have offices around the UK however, so working from those is more than possible too. DVELP has large clients all around the country, so occasional travel to their offices may be required.
```

Comment:
`DVELP has large clients all around Europe, so occasional travel to their offices may be required.`